sdk/storage/storage-blob/package.json

```diff
@@ -129,7 +129,6 @@
     "tslib": "^2.0.0"
   },
   "devDependencies": {
-    "@azure/core-rest-pipeline": "1.0.0-beta.2",
```

Comment:
No longer used after perf tests were moved out.

Reply:
@jeremymeng 
`"@azure/test-utils-perfstress"` a few lines below.. can also be removed.

Reply:
yes. although I don't like resetting CI builds, maybe in another PR?